modules/vpc/versions.tf

```diff
@@ -8,7 +8,7 @@ terraform {
     }
 
     google-beta = {
-      source  = "hashicorp/google"
+      source  = "hashicorp/google-beta"
```

Comment:
I dont see the google-beta provider being used in the code. Please remove this if not necessary.